bin/kick_off.sh

```
@@ -164,19 +164,27 @@ tests=0
 #
 if [[ $ssh_key_file != "" ]]; then
-	cp $ssh_key_file config/user.pem_test
+	if [ ! -f $ssh_key_file ]; then
+		echo "${ssh_key_file} does not exists, please fix.  Test is exiting"
+		exit 1
+	fi
 	if [ ! -s $ssh_key_file ]; then
 		echo "${ssh_key_file} is zero length, please fix.  Test is exiting"
 		exit 1
 	fi
+	cp $ssh_key_file config/user.pem_test
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)